composer.json

```diff
@@ -5,19 +5,18 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
-        "sylius/sylius": "~1.3.0@dev"
+        "sylius/sylius": "~1.4.0"
```

Comment:
```suggestion
        "sylius/sylius": "^1.3.0"
```

Reply:
I need tu change it too ?
https://github.com/loevgaard/SyliusBrandPlugin/pull/33/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R58